decdk.specs.json

```diff
@@ -0,0 +1,1070 @@
+{
```

Comment:
We should create a Json schema for this spec and version it accordingly, similar to what is done for the CDK [cloud assembly schema][1]. This will allow the spec and consumers to evolve semi-independently without breaking changes (or, at least, fail fast in case the consumer can't process some particular version).

[1]: https://github.com/aws/aws-cdk/tree/main/packages/%40aws-cdk/cloud-assembly-schema

Reply:
Ok, I've done that already.